pandas/tests/io/parser/dtypes/test_dtypes_basic.py

```diff
@@ -204,6 +204,11 @@ def test_1000_sep_decimal_float_precision(
     # test decimal and thousand sep handling in across 'float_precision'
     # parsers
     decimal_number_check(c_parser_only, numeric_decimal, thousands, float_precision)
+    text, value = numeric_decimal
```

Comment:
can you add a test exactly like the OP: https://github.com/pandas-dev/pandas/issues/43713 as well